addons/hw_drivers/iot_handlers/drivers/PrinterDriver_L.py

```diff
@@ -391,6 +394,34 @@ def print_status(self):
         title = commands['title'] % b'IoTBox Status'
         self.print_raw(commands['center'] + title + b'\n' + wlan.encode() + mac.encode() + ip.encode() + homepage.encode() + pairing_code.encode() + commands['cut'])
 
+    def print_status_zpl(self):
+        mac = helpers.get_mac_address()
+        pairing_code = connection_manager.pairing_code
+        ssid = wifi.get_access_point_ssid() if wifi.is_access_point() else wifi.get_current()
+
+        interfaces = ni.interfaces()
+        ips = []
+        for iface_id in interfaces:
+            iface_obj = ni.ifaddresses(iface_id)
+            ifconfigs = iface_obj.get(ni.AF_INET, [])
+            for conf in ifconfigs:
+                if 'addr' in conf and conf['addr'] not in ['127.0.0.1', '10.11.12.1']:
+                    ips.append(conf.get('addr'))
```

Comment:
```suggestion
                if 'addr' in conf and conf['addr'] not in ['127.0.0.1', '10.11.12.1']:
                    ips.append(conf['addr'])
```
If the condition did not return a traceback (KeyError), then there is no risk inside of it